packages/amplify-provider-awscloudformation/src/setup-new-user.js

```diff
@@ -6,6 +6,7 @@ const constants = require('./constants.js');
 const systemConfigManager = require('./system-config-manager');
 const obfuscationUtil = require('./utility-obfuscate');
 const { open } = require('amplify-cli-core');
+const isOnWsl = require('is-wsl');
```

Comment:
This dependency needs to be added to `packages/amplify-provider-awscloudformation/package.json`

Reply:
Added. Thanks for the review.